turret-hosts/reference-implementation.md

```diff
@@ -0,0 +1,216 @@
+# Stellar Turrets Reference Implementation
```

Comment:
Nice!

A cool addition would be some of the details in https://github.com/tyvdh/stellar-turrets/pull/36 about running more than one turret

Reply:
Good call! Maybe I'll add some notes about testnet vs mainnet with a link to Tyler's PR, since it's still a work-in-progress?